api/taxonomies/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from api.taxonomies import views
+
+urlpatterns = [
+    url(r'^plos/flat$', views.PlosTaxonomyFlat.as_view(), name=views.PlosTaxonomyFlat.view_name),
```

Comment:
Add a / to the end of these urlpatterns (before the $) -- as it stands, the URL won't resolve if you go to `v2/taxonomies/plos/` , but with the extra slash in the URL regex, both `v2/taxonomies/plos/` and `v2/taxonomies/plos` will work just fine